cart.go

```diff
@@ -495,3 +473,31 @@ func mutateURL(original string, mutate bool) string {
 
 	return safe.String()
 }
+
+func httpClient(url, token string) []byte {
```

Comment:
This should be `httpGet` or similar. The name `httpClient` implies it returns an `http.Client`, rather than performing a full request cycle.

Reply:
Thanks for the review, fixed!